File: CP/code/intro/uva_11942.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        try (Scanner sin = new Scanner(System.in)) {
            int N = sin.nextInt();

            System.out.println("Lumberjacks:");
            while (N-- > 0) {
                boolean ascend = true, descend = true;

                int prev = sin.nextInt();
                for (int i = 0; i < 9; i++) {
                    int cur = sin.nextInt();

                    if (cur > prev)
                        descend = false;
                    else if (cur < prev)
                        ascend = false;

                    prev = cur;
                }

                System.out.println(ascend || descend ? "Ordered" : "Unordered");
            }
        }
        
    }    
}

File: CP/code/intro/uva_11942.cc
#include <cstdio>

using namespace std;

int main(int argc, char* argv[]) 
{
    int N;
    scanf("%d", &N);
    
    printf("Lumberjacks:\n");
    while (N--) {
        bool ascend = true;
        bool descend = true;

        int prev;
        scanf("%d", &prev);
        for (int i = 1; i < 10; i++) {
            int cur;
            scanf("%d", &cur);

            if (cur > prev)
                descend = false;
            else if (cur < prev)
                ascend = false;
            
            prev = cur;
        }

        printf(descend || ascend ? "Ordered\n" : "Unordered\n");
    }

    
    return 0;
}